### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by whitneykidd**
Updated variable names and tests as discussed and wrote attributes test for game_team_stats_test

### CODE REVIEW

File: test/game_stats_test.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ def test_game_objects_are_new_instances_of_game_class
     assert_instance_of Game, @games_stats.games.first
   end
 
-  def test_attributes_for_instance_of_game_withing_game_stats
```
Comment (tylertomlinson) [inline test/game_stats_test.rb:21]: Good catch™. My bad

File: test/game_teams_stats_test.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  def setup
+    @game_team_stats = GameTeamStats.new("./data/game_teams_truncated.csv", GameTeams)
+    @game_teams = @game_team_stats.game_teams[1]
+  end
+
+  def test_it_exists
+    assert_instance_of GameTeamStats, @game_team_stats
+  end
+
```
Comment (tylertomlinson) [inline test/game_teams_stats_test.rb:16]: Now SimpleCov will cover each GameTeams

File: test/team_stats_test.rb [hunk 3]
```diff
@@ -6,12 +6,18 @@
 class TeamStatsTest < Minitest::Test
 
   def setup
-    @team_stats = TeamStats.new("./data/teams.csv", TeamStats)
```
Comment (tylertomlinson) [inline test/team_stats_test.rb:9]: Another good catch™

File: lib/game_teams.rb [hunk 4]
```diff
@@ -3,22 +3,22 @@ class GameTeams
...
+    @game_id = attributes[:game_id].to_i
+    @team_id = attributes[:team_id].to_i
+    @hoa = attributes[:hoa]
+    @result = attributes[:result]
     @settled_in = attributes[:settled_in]
-    @head_couch = attributes[:head_couch]
-    @goals = attributes[:goals]
-    @shots = attributes[:shots]
-    @tackles = attributes[:tackles]
+    @head_coach = attributes[:head_coach]
```
Comment (Maxwell-Baird) [inline lib/game_teams.rb:19]: good job catching my mistakes

File: lib/game_teams.rb [hunk 5]
```diff
@@ -3,22 +3,22 @@ class GameTeams
   attr_reader :game_id,
               :team_id,
               :hoa,
-              :results,
+              :result,
               :settled_in,
-              :head_couch,
```
Comment (alex-latham) [inline lib/game_teams.rb:8]: LOL

File: test/team_stats_test.rb (hunk 3 above)
Comment (alex-latham) [reply test/team_stats_test.rb:9]: Thanks for finding that - I introduced this bad line to the code (sorry),.

**Pull request merged**